### PR TITLE
Remove broken f2c files: build 1002

### PR DESCRIPTION
**Pull request by Areustle**
The build 1002 does not contain the f2c executable and is superseded by build 1004, as merged by https://github.com/conda-forge/f2c-feedstock/pull/13

Guidelines for marking packages as broken:

* We prefer to patch the repo data (see [here](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock))
  instead of marking packages as broken. This alternative workflow makes environments more reproducible.
* Packages with requirements/metadata that are too strict but otherwise work are
  not technically broken and should not be marked as such.
* Packages with missing metadata can be marked as broken on a temporary basis
  but should be patched in the repo data and be marked unbroken later.
* In some cases where the number of users of a package is small or it is used by
  the maintainers only, we can allow packages to be marked broken more liberally.
* We (`conda-forge/core`) try to make a decision on these requests within 24 hours.

What will happen when a package is marked broken?

* Our bots will add the `broken` label to the package. The `main` label will remain on the package and this is normal.
* Our bots will rebuild our repodata pacthes to remove this package from the repodata.
* In a few hours after the `anaconda.org` CDN picks up the new patches, you will no longer be able to install the package from the `main` channel.

Checklist:

* [x] Make sure your package is in the right spot (`broken/*` for adding the
  `broken` label, `not_broken/*` for removing the `broken` label, or `token_reset/*`
  for token resets)
* [x] Added a description of the problem with the package in the PR description.
* [x] Added links to any relevant issues/PRs in the PR description.
* [x] Pinged the team for the package for their input.

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks Alex! 🙏

Should 1003 be removed too?

File: broken/example.txt [hunk 1]
```diff
@@ -1,2 +1,7 @@
 win-64/cf-autotick-bot-test-package-0.4-py38_0.tar.bz2
 win-64/cf-autotick-bot-test-package-0.4-py27_0.tar.bz2
+osx-arm64/f2c-20210323183018-h1c322ee_1002.tar.bz2
+osx-64/f2c-20210323183018-h5eb16cf_1002.tar.bz2
+linux-ppc64le/f2c-20210323183018-hb283c62_1002.tar.bz2
+linux-aarch64/f2c-20210323183018-h4e544f5_1002.tar.bz2
+linux-64/f2c-20210323183018-h166bdaf_1002.tar.bz2
```
Comment (jakirkham) [inline broken/example.txt:7]: Also these should be added to a new file in the same directory

**Comment by Areustle** [comment]
Build 1003 was not committed as far as I can tell. But if you see an instance of it then yes, remove it.

File: broken/example.txt (hunk 1 above)
Comment (Areustle) [reply broken/example.txt:7]: You're right. It was late for me when I opened this PR.

Comment (jakirkham) [reply broken/example.txt]: No worries. Thanks for fixing that

**Comment by jakirkham** [comment]
Thanks Alex! 😄

**Pull request merged**